package.json

```
@@ -38,6 +38,7 @@
     "prepublishOnly": "echo 'To publish this, go to build/ and run 'yarn publish''",
     "watch:test": "yarn test --watch",
     "test": "NODE_ENV=test jest src/ scripts/",
+    "test:single": "NODE_ENV=test jest",
```

What is this used for?

---

I added this to test a single file.
If i do`yarn test Form.test` all tests are running.
With `yarn test:single Form.test`i can test just the `Form.test.js`

---

```suggestion
    "jest": "NODE_ENV=test jest",
```

What about that? Executes the following for me:

>$ yarn jest src/molecules/FloatingButton.test.js 
>$ NODE_ENV=test jest src/molecules/FloatingButton.test.js

---

Or I think better would be:

```
"test": "NODE_ENV=test jest",
"test:all": "yarn test src/ scripts/"
```

Then you could still use `yarn test src/MyFile.jsx`

---

Think i like it.
So `yarn test` still behaves the same and `yarn jest Form.test` would just run the given test (if provided). Right?